.travis.yml

```diff
@@ -10,9 +10,9 @@ env:
   global:
     - COMPOSER_ARGS="--no-interaction"
     - COVERAGE_DEPS="satooshi/php-coveralls"
-    - TESTS_ZEND_DB_ADAPTER_DRIVER_MYSQL_HOSTNAME=localhost
-    - TESTS_ZEND_DB_ADAPTER_DRIVER_MYSQL_PASSWORD=
-    - TESTS_ZEND_DB_ADAPTER_DRIVER_PGSQL_HOSTNAME=localhost
+
+services:
+  - mysql
```

Comment:
Move services to individual configuration blocks.
This will allow:
1. Ability to configure specific multiple supported versions. Eg, PgSQL 10 is not BC with 9.6.
2. Prevent travis from running multiple conflicting engines/configurations per build. Harder to see what version failed, unknown consequences.